tests/unit/test_mesmer_x_distrib_cov.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+import pytest
+
+from mesmer.mesmer_x import Expression, distrib_cov
+
+
+def test_distrib_cov_init_all_default():
+    rng = np.random.default_rng(0)
+    n = 250
+    pred = np.linspace(0, 1, n)
+    targ = rng.normal(loc=2 * pred, scale=0.1, size=n)
+
+    expression = Expression("norm(loc=c1 * __tas__, scale=c2)", expr_name="exp1")
+
+    dist = distrib_cov(targ, {"tas": pred}, expression)
+
+    np.testing.assert_equal(dist.data_targ, targ)
+    np.testing.assert_equal(dist.data_pred, {"tas": pred})
+    np.testing.assert_equal(dist.weights_driver, np.ones(n) / n)
+    assert dist.n_sample == n
+    assert dist.expr_fit == expression
```

Comment:
Does that actually work? Ah it does but I think it checks that it's the same objects ant thus better to use `is` to be more explicit.
```suggestion
    assert dist.expr_fit is expression
```

Reply:
Ah thanks!